L/libnovas/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libnovas"
+version = v"3.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/kiranshila/novas.git", "0d4d08650d3b8a0f798da2b86f894b5083654e32")
```

Comment:
I'd prefer using an upstream source instead of a personal fork, if possible.  If you want to add extra files, you can add them to a subdirectory called, e.g., `bundled` and use a [`DirectorySource`](https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.DirectorySource)